weave/type_handlers/__init__.py

```diff
@@ -0,0 +1,5 @@
+from weave.type_handlers.Audio import audio
+from weave.type_handlers.Image import image
+
+image.register()
+audio.register()
```

Comment:
why not keep this pattern for registering preparers?

Reply:
yeah I guess we could put it here -- will change

Reply:
changed